dataclients/kubernetes/ingressv1.go

```diff
@@ -78,6 +79,17 @@ func convertPathRuleV1(
 		}
 	} else if svc.Spec.Type == "ExternalName" {
 		return externalNameRoute(ns, name, host, hostRegexp, svc, servicePort, allowedExternalNames)
+	} else if forceKubernetesService {
+		r := &eskip.Route{
+			Id:          routeID(ns, name, host, prule.Path, svcName),
+			BackendType: eskip.NetworkBackend,
+			Backend:     serviceNameBackend(svcName, ns, servicePort),
+			HostRegexps: hostRegexp,
+		}
+
+		setPathV1(pathMode, r, prule.PathType, prule.Path)
+		setTraffic(r, svcName, prule.Backend.Traffic, prule.Backend.NoopCount)
+		return r, nil
```

Comment:
This looks the same as a special case `if len(eps) == 1` below. 
Wouldn't it be better to just have `eps = []string{serviceNameBackend(svcName, ns, servicePort)}` here?

Reply:
I guess we'd also need to have it for RouteGroups around here https://github.com/zalando/skipper/blob/d3e8dbd82527da74ac3fb4f4a9868a840478b81f/dataclients/kubernetes/routegroup.go#L241

Reply:
@szuecs Do we need this for the `RouteGroup` or maybe users should explicitly use `network` backend (then this needs to be mentioned in the docs I guess)?

Reply:
> I don't fully understand your comments around RouteGroups

RouteGroup is a Skipper CRD and could also use Service domain name but on the other hand users have much more control over backend configuration so they may use `network` backend and Service domain name explicitly therefore I think we do not need to support this new config there.

Reply:
@szuecs Do we want this for [ingressv1beta1](https://github.com/zalando/skipper/blob/37245e82ade1d1aab359443e73d5fc0f966ade56/dataclients/kubernetes/ingressv1beta1.go#L103-L104) for consistency? I know that we plan to drop it https://github.com/zalando/skipper/pull/1999

Reply:
> @szuecs Do we want this for [ingressv1beta1](https://github.com/zalando/skipper/blob/37245e82ade1d1aab359443e73d5fc0f966ade56/dataclients/kubernetes/ingressv1beta1.go#L103-L104) for consistency? I know that we plan to drop it #1999

I purposely left it out for two reasons:
- I can't imagine this feature is interesting for active users (why switch to services if the current setup works)
- I assumed the group of users that would like to start using ingresses today (but can't due to the default usage of Endpoints) are still rocking an old Kubernetes version, V1 is out for quite a while already right?

Reply:
I think:
- RouteGroup can use a service with DNS via network backend. In case there is a need we can add it
- v1beta1 I would not touch and we should clean this up after our internal migration (1year deprecation seems to be also fine for everyone else)